<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>team_editor</class>
 <widget class="QDialog" name="team_editor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Team Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="page_title">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Team Editor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="lineWidth">
        <number>6</number>
       </property>
       <property name="midLineWidth">
        <number>4</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="list_title">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Members on Team</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="member_list"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="member_delete_title">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Update or Delete Member</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="midLineWidth">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_member_button">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Delete Member</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="member_update_title">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Update Member Info:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="midLineWidth">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="update_member_name_label">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Update Member Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="update_member_name_line_edit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="update_member_email_label">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Update Member Email:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="update_member_email_line_edit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="update_member_button">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Update Member</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="add_team_title">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add New Member to Team</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="add_member_name_label">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Member Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="add_member_name_line_edit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="add_member_email_label">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Member Email:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="add_member_email_line_edit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="add_member_button">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add Member</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>team_editor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>team_editor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
